Code/Framework/AzFramework/AzFramework/Archive/ArchiveVars.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/base.h>
+#include <AzFramework/Archive/ArchiveVars.h>
+
+namespace AZ::IO
+{
+    FileSearchPriority GetDefaultFileSearchPriority()
+    {
+#if defined(O3DE_DEFAULT_FILE_SEARCH_MODE)
```

Comment:
where is this defined? I imagine you want to pass this from cmake and have it as a cache variable

Reply:
Ah this change, shouldn't be in this PR